examples/fastrp-and-knn.ipynb

```diff
@@ -379,27 +386,8 @@
   }
  ],
  "metadata": {
-  "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
-   "language": "python",
-   "name": "python3"
-  },
   "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.8.10"
-  },
-  "vscode": {
-   "interpreter": {
-    "hash": "916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1"
-   }
```

Comment:
nice you could get rid of these. Is there a way to avoid this in the future.
I think vs code added these automatically - e.g. what did you run to clean it?

Reply:
There's a tool called `jupyter nbconvert` that one can use. Could be nice to add this to `makestyle`/`checkstyle` actually

Reply:
Thats also what nicola uses to convert them to docs :+1:

Reply:
ah ok, cool :)

Reply:
I can make a PR for that as well then

Reply:
Yes, this depends on the tools we use to created/edit notebooks. I don't know if they could be useful for something, but at the moment I am not against scrubbing them via styling if possible. If a tool reintroduces them, we should look into how to configure the tool as well (for instance there should be such an option in VSCode).